OOModel/src/expressions/MetaCallExpression.cpp

```diff
@@ -62,4 +69,61 @@ MetaDefinition* MetaCallExpression::metaDefinition()
 	return ret;
 }
 
+void MetaCallExpression::bindMetaCalls(Model::Node* n, MetaBinding* binding)
+{
+	if (auto metaCall = DCast<MetaCallExpression>(n))
+		if (auto callee = DCast<ReferenceExpression>(metaCall->callee()))
+			for (auto i = 0; i < binding->mappings()->size(); i++)
+				if (callee->name() == binding->mappings()->at(i)->name())
+					callee->setName(binding->mappings()->at(i)->value()->name());
```

Comment:
In the current implementation the modified trees are not attached to the main tree. Therefore it is not necessary.